### ISSUE TITLE
A11y_SQL Azure Data Studio_NewTables_Cloumns_Info & Relationship: NVDA is not announcing the Alert error popup screen completely.

### ISSUE THREAD

**Issue by ShivamPandey-V** [microsoft/azuredatastudio#19413]
**GitHub Tags:**
#A11yMAS;#A11yTCS;#Win11;#WCAG1.3.1;#DesktopApp;#A11ySev3;#SH_SQL Azure Data Studio_May2022;#SQL Azure Data Studio;

## Environmental Details:
VS Code: 1.59.0
Electron: 13.6.6
Chrome: 91.0.4472.164
Node.js: 14.16.0
V8: 9.1.269.39-electron.0

## OS Details :
Microsoft Windows 11

## Repro Steps:
1.Add connection to the server with appropriate credentials.
2.Tab till Servers in left navigation Bar.
3.Tab till tables and Press enter
4.Tab till New tables and press enter
5.Tab till column and press enter.
6.turn on the NVDA.
6.rename the table name and keep it as blank and press tab error popup will appear verify popup is announced by nvda or not.

## Actual results:
NVDA is not announcing the Alert error popup screen completely.

## Expected results:
NVDA should announce the Alert error popup screen completely.

## User Impact:
Screen reader users will not be able to understand the Alert if it is not announced completely.

## Attachments:
https://user-images.githubusercontent.com/100270292/169009298-1a9e3c98-7459-42d2-b416-f34e0c8f1227.mp4
